com.theappgineer.community_remote.yml

```diff
@@ -0,0 +1,29 @@
+id: com.theappgineer.community_remote
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: community_remote
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-documents
```

Comment:
why do you need that filesystem permissions?

Reply:
It might be too much, I'm new to flatpak. What I need is write access for a configuration file. Can this be dealt with without any file system permission? I will do some more reading and testing on this topic.

Reply:
The configuration file lives in `~/.var/app/{id}/data`. If there is no extra permission required to write to that location, and it is persistent between updates, then I will remove the `xdg-documents` permission.

Reply:
if you do that outside of flatpak it's totally wrong. Inside flatpak that where the XDG dirs are redirected automatically.

xdg-documents would definitely be wrong either way.

Reply:
It is all inside the flatpak, I removed the filesystem permission.